package.json

```diff
@@ -49,10 +52,12 @@
     "benchr": "^3.2.0",
     "coffee-script": "^1.10.0",
     "doctoc": "^1.3.0",
+    "eslint": "^4.8.0",
     "expect": "^1.20.2",
     "microtime": "^2.1.2",
     "mocha": "^2.5.3",
     "moo": "^0.3.2",
+    "prettier": "^1.7.4",
```

Comment:
It is better to use a fixed version number of prettier without a wildcard ('^') because
minor changes may cause formatting changes which will fail the CI build (when using --list-different). 

See: https://github.com/prettier/prettier#usage for details (at end of this section).